pkg/graphviz/traceflow_test.go

```diff
@@ -369,3 +600,11 @@ func TestGetTraceflowStatusMessage(t *testing.T) {
 		})
 	}
 }
+
+func TestFindClusterString(t *testing.T) {
```

Comment:
More test cases can be added to improve the coverage of this function, I think still there is a scope.

Reply:
All other cases of this function is already covered in other function so leftover case is covered

Reply:
You can add a comment about it.